packages/webapp/src/pages/Team/Billing/Show.tsx

```diff
@@ -0,0 +1,134 @@
+                </Helmet>
+                <h2 className="text-3xl font-semibold text-white mb-16">Billing</h2>
+                <div className="flex flex-col gap-4">
+                    <Skeleton className="w-[250px]" />
+                    <Skeleton className="w-[250px]" />
+                </div>
+            </DashboardLayout>
+        );
+    }
+
+    if (error) {
+        return <ErrorPageComponent title="Billing" error={error} page={LeftNavBarItems.TeamBilling} />;
+    }
+
+    if (!currentPlan) {
+        return null;
+    }
+
+    return (
+        <DashboardLayout selectedItem={LeftNavBarItems.TeamBilling}>
+            <Helmet>
+                <title>Billing - Nango</title>
+            </Helmet>
+            <h2 className="text-3xl font-semibold text-white mb-16">Billing</h2>
+            <div className="flex flex-col gap-4">
+                <div className="flex flex-col gap-2.5">
+                    <h2 className="text-grayscale-10">Plan</h2>
+                    <div className="grid grid-cols-3 gap-4">
+                        {plans.map((def) => {
+                            return <PlanCard key={def.plan.code} env={env} def={def} />;
+                        })}
+                    </div>
+
+                    <div className="flex text-white text-sm">
+                        <Link to="https://nango.dev/pricing" target="_blank" className="flex gap-2">
+                            <IconExternalLink stroke={1} size={18} />
+                            View pricing page
+                        </Link>
+                    </div>
+                </div>
+            </div>
+        </DashboardLayout>
+    );
+};
+
+export const PlanCard: React.FC<{ env: string; def: PlanDefinitionList }> = ({ def }) => {
+    return (
+        <Link
+            className={cn(
+                'flex flex-col gap-4 text-white rounded-lg bg-grayscale-3 py-7 px-6 border border-grayscale-5',
+                def.active && 'bg-grayscale-1 border-grayscale-7'
+            )}
+            target="_blank"
+            to={'https://nango.dev/pricing'}
+            // to={def.above && !def.plan.canUpgrade ? 'mailto:upgrade@nango.dev' : ''}
+            // onClick={onClickPlan}
+        >
+            <div className="flex flex-col gap-2.5">
+                <header className="flex gap-3 items-center">
+                    <div className="capitalize">{def.plan.title}</div>
+                    {def.active && <div className="bg-success-4 h-1.5 w-1.5 rounded-full"></div>}
+                </header>
+                <div className="text-sm text-grayscale-10">{def.plan.description}</div>
+            </div>
+            {/* <footer>
+                {!def.active && (
+                    <Button variant={'primary'} isLoading={loading}>
+                        {def.plan.cta ? def.plan.cta : def.below ? 'Downgrade Plan' : 'Upgrade plan'}
+                    </Button>
+                )}
+            </footer> */}
```

Comment:
I imagine this is going to be used at  some point

Reply:
yes didn't want to lose this :D